test/workers/listener.js

```diff
@@ -0,0 +1,280 @@
+// Modules
+import {describe, it, before, after} from 'mocha';
+import {expect} from 'chai';
+
+// Mocks
+import * as mockery from '../mocks/mockery.js';
+
+// Target
+import listener from '../../src/workers/listener.js';
+import slackBot from '../../src/workers/slack-bot.js';
+
+// Tests
+describe('listenerTest', () => {
+
+    const output = [];
+    const input = [];
+
+    before(() => {
+        mockery.setup({
+            output: output,
+            input: input
+        });
+    });
+
+    it('Should initialize the listener', () => {
+        const myListener = listener({bot: slackBot()});
+        myListener.listen();
+        expect(myListener.listening).to.equal(true);
```

Comment:
this really feels like true === true, maybe improve the truthiness of the listener.listen function :-)